backend/src/services/mural.service.ts

```diff
@@ -3,7 +3,16 @@ import { UpdateOptions } from "sequelize";
 
 export class MuralService {
   public async create(mural: MuralInterface) {
-    const createdMural: Mural = await Mural.create<Mural>(mural);
+    const coordinates = {
+      coordinates: {
+        type: "Point",
+        coordinates: [mural.longitude, mural.latitude],
+      },
+    };
+    const createdMural: Mural = await Mural.create<Mural>({
+      ...mural,
+      ...coordinates,
```

Comment:
the ... lets us pass a new object that has the attributes of mural and of coordinates, so essentially 'mural' with an added attribute:
 coordinates: {
        type: "Point",
        coordinates: [mural.longitude, mural.latitude],
      }